exporters/committime/collector_base.py

```diff
@@ -14,14 +14,16 @@ class AbstractCommitCollector(pelorus.AbstractPelorusExporter):
     This class should be extended for the system which contains the commit information.
     """
 
-    def __init__(self, kube_client, username, token, namespaces, apps, collector_name, timedate_format, git_api=None):
+    def __init__(self, kube_client, username, token, namespaces, apps, collector_name,
+                 timedate_format, git_api=None, tls_verify=None):
```

Comment:
I think we said we wanted to generalize `git_api` to something like `api_url`, right?

As much as I hate to bikeshed, changing parameter names in python is a breaking change because of keyword arguments. Not that it matters when we're the only consumer 😛 

Also, have we tested what happens with `verify=None`? The [requests](https://docs.python-requests.org/en/master/api/#requests.request) docs say it has to be a boolean or string. I think we should have it be `tls_verify=True` just to be safe.